boards/riscv/rv32m1_vega/doc/index.rst

```diff
@@ -116,6 +116,44 @@ supports the following hardware features:
 |           |            | fxos8700 trigger;                   |
 +-----------+------------+-------------------------------------+
 
+BLE Software Link Layer experimental support
+==================================================
+This is an experimental feature supported on the Zephyr's RI5CY
+configuration, ``rv32m1_vega_ri5cy``. It  uses the Software Link Layer
+framework by Nordic Semi to enable the usage of the on-SoC radio and
+transceiver for implementing of a software defined BLE controller.
+By using both the controller and the host stack available in Zephyr, the
+following BLE samples can be used with this board:
+
+- beacon
+- central
+- central_hr
+- eddystone
+- hci_uart
+- ibeacon
+- peripheral_csc (Cycling Speed Cadence)
+- peripheral_dis (Device Information Service)
+- peripheral_esp (Environmental Sensing Service)
+- peripheral_hr (Heart Rate)
+- peripheral_ht (Health Thermometer)
+- scan_adv
+
+.. warning::
```

Comment:
This feels more like a "note" than a "warning" (nothing of potential harm or damage is described here).

Reply:
Correct, the logic behind putting it as a 'warning' instead of 'note' was because some of the limitations are unexpected let's say (f.i. controller-based encryption). But you are right, nothing of potential harm is described, and as such I will make it a "note" and remove the previous "note" directive.